build-system/tasks/performance/helpers.js

```diff
@@ -19,13 +19,16 @@ const fs = require('fs');
 const path = require('path');
 
 const CDN_URL = 'https://cdn.ampproject.org/';
+const V0_PATH = '/dist/v0.js';
+const LOCAL_PATH_REGEXP = /dist\/(v0\/amp-[A-Za-z\-0-9\.]+.js)/;
+const ANALYTICS_VENDORS_PATH = '../../../extensions/amp-analytics/0.1/vendors/';
```

Comment:
Let's use the compiled vendor which lives under v0/analytics-vendors/xxx.json

Reply:
Ok, if we're using that path then we should also add the `gulp vendor-configs` command as part of this task...

That command will be run in the Travis build, but I think it's good to have it for local testing.

Reply:
I believe we always run that with `gulp build` or `gulp dist`, so it should be fine?